src/status_im/utils/handlers.clj

```diff
@@ -0,0 +1,12 @@
+(ns status-im.utils.handlers)
+
+(defmacro handlers->
```

Comment:
I agree with your sentiment. On the other hand chaining those functions manually is a little tricky and error prone are pure side-effect while others mutate db.
I expect this usage to significantly decrease after we introduce more `interceptors` and `reg-fx` usages.

Consider this as intermediary bandage to help migrating to better patterns :)